jsons/parameters.json

```diff
@@ -16703,5 +16703,20 @@
     "triggers_rerun": true,
     "value_translations": {},
     "value_type": "int"
+  },
+  {
+    "default_value": null,
+    "description": "pandas Query string to be applied to filter a dataframe",
+    "key_translations": {
+      "csv_filter_style_1": "query",
+      "ursgal_style_1": "query_string"
```

Comment:
shouldn't be that the other way around? the csv_filter takes query_string.

Reply:
I updated it to pandas_query_string for ursgal_style_1 and query_string for csv_filter_style_1. I think it would be more explicit as such, in case one day we might want to have a param e.g. "sql_query_string" or so :)